docs/openapi.yaml

```diff
@@ -0,0 +1,183 @@
+openapi: 3.0.3
+
+info:
+  title: NVA Download API
+  description: >
+    The NVA Download API.
+  termsOfService: https://nva.unit.no/terms/
+  contact:
+    email: nva@unit.no
+  license:
+    name: MIT/NLOD 2.0
+    url: https://link.to.license
+  version: 1.0.0
+
+externalDocs:
+  description: Find out more about NVA
+  url: https://nva.unit.no
+
+paths:
+  /{identifier}/files/{fileIdentifier}:
+    get:
+      summary: Returns an S3 presigned download URL to a publication file (within json response body)
```

Comment:
Can you shorten down this to about half the length? You can add the full length text as a "description" instead.